src/modules/communication/internals/send-grid-service.ts

```diff
@@ -0,0 +1,32 @@
+import { SendEmailParams } from '../types';
+import config from 'config';
+import mail from '@sendgrid/mail';
+
+export default class SendGridService {
+  public static initializeService() {
+    const sendgridAPIKey:string = config.get('sendgridAPIKey'); // Todo: here use configuration service to get these details.
+    if(sendgridAPIKey) {
```

Comment:
This does not sound right, if the caller has asked SendGridService.initializeService and if config is not, it should throw.

Failing silently results in unexpected behaviours.